redwood-composeui/src/commonMain/kotlin/app/cash/redwood/composeui/RedwoodContent.kt

```diff
@@ -31,13 +46,36 @@ public fun RedwoodContent(
 ) {
   val scope = rememberCoroutineScope()
   val children = remember(provider, content) { ComposeWidgetChildren() }
+
+  var viewportSize by remember { mutableStateOf(Size.Zero) }
+  val density = LocalDensity.current
+  val uiConfiguration = UiConfiguration(
+    darkMode = isSystemInDarkTheme(),
+    safeAreaInsets = safeAreaInsets(),
```

Comment:
Hmm shouldn't the insets code have moved into this module rather than into a new `widgets-composeui`?

Reply:
Done as part of https://github.com/cashapp/redwood/pull/1459